satellite/accounting/tally/tally.go

```diff
@@ -31,21 +32,19 @@ type Service struct {
 	logger                  *zap.Logger
 	metainfo                *metainfo.Service
 	overlay                 *overlay.Service
-	limit                   int
-	ticker                  *time.Ticker
+	Loop                    sync2.Cycle
 	storagenodeAccountingDB accounting.StoragenodeAccounting
 	projectAccountingDB     accounting.ProjectAccounting
 	liveAccounting          live.Service
 }
 
 // New creates a new tally Service
-func New(logger *zap.Logger, sdb accounting.StoragenodeAccounting, pdb accounting.ProjectAccounting, liveAccounting live.Service, metainfo *metainfo.Service, overlay *overlay.Service, limit int, interval time.Duration) *Service {
+func New(logger *zap.Logger, sdb accounting.StoragenodeAccounting, pdb accounting.ProjectAccounting, liveAccounting live.Service, metainfo *metainfo.Service, overlay *overlay.Service, interval time.Duration) *Service {
 	return &Service{
 		logger:                  logger,
 		metainfo:                metainfo,
 		overlay:                 overlay,
-		limit:                   limit,
```

Comment:
Was this not used anywhere? At least i couldnt find an change here, that used it.

Reply:
Yeah, it was unused.